imports/pages/video/wowza.js

```diff
@@ -0,0 +1,52 @@
+import { Component, PropTypes } from "react";
+import { connect } from "react-redux";
+import { css } from "aphrodite";
+import ReactJWPlayer from "react-jw-player";
+import navActions from "../../data/store/nav";
+import headerActions from "../../data/store/header";
+import styles from "../../components/@primitives/nav/offset-css";
+
+// eslint-disable-next-line
+class TemplateWithoutData extends Component {
+
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+  }
+
+  componentWillMount() {
+    this.props.dispatch(navActions.setLevel("BASIC_CONTENT"));
+    this.props.dispatch(headerActions.set({
+      title: "Wowza",
+    }));
+  }
+
+  render() {
+    return (
+      <div className={`locked-ends locked-sides background--dark-primary floating ${css(styles.offset)}`}>
+        <div className="floating__item one-whole">
+          <ReactJWPlayer
```

Comment:
Yea, we should look at this component more closely. It appears to be only cloud jwplayer based rather than local player. If we plan on using cloudfront + wowza but might need to customize a jwplayer component to use a self-hosted player. We could even add support for a certain "native" framework. 😉